sdk/batch/arm-batch/CHANGELOG.md

```diff
@@ -1,15 +1,33 @@
 # Release History
+    
+## 9.0.0 (2023-06-30)
+    
+**Features**
 
-## 8.0.1 (Unreleased)
-
-### Features Added
-
-### Breaking Changes
+  - Added Type Alias ContainerType
+  - Interface NetworkConfiguration has a new optional parameter enableAcceleratedNetworking
+  - Interface VMExtension has a new optional parameter enableAutomaticUpgrade
+  - Added Enum KnownContainerType
+  - Interface ApplicationListNextOptionalParams no longer has parameter maxresults
+  - Interface ApplicationPackageListNextOptionalParams no longer has parameter maxresults
+  - Interface CertificateListByBatchAccountNextOptionalParams no longer has parameter filter
+  - Interface CertificateListByBatchAccountNextOptionalParams no longer has parameter maxresults
+  - Interface CertificateListByBatchAccountNextOptionalParams no longer has parameter select
+  - Interface LocationListSupportedCloudServiceSkusNextOptionalParams no longer has parameter filter
+  - Interface LocationListSupportedCloudServiceSkusNextOptionalParams no longer has parameter maxresults
+  - Interface LocationListSupportedVirtualMachineSkusNextOptionalParams no longer has parameter filter
+  - Interface LocationListSupportedVirtualMachineSkusNextOptionalParams no longer has parameter maxresults
+  - Interface PoolListByBatchAccountNextOptionalParams no longer has parameter filter
+  - Interface PoolListByBatchAccountNextOptionalParams no longer has parameter maxresults
+  - Interface PoolListByBatchAccountNextOptionalParams no longer has parameter select
+  - Interface PrivateEndpointConnectionListByBatchAccountNextOptionalParams no longer has parameter maxresults
+  - Interface PrivateLinkResourceListByBatchAccountNextOptionalParams no longer has parameter maxresults
 
-### Bugs Fixed
+**Breaking Changes**
 
-### Other Changes
+  - Type of parameter type of interface ContainerConfiguration is changed from "DockerCompatible" to ContainerType
```

Comment:
@kazrael2119 Could you help refer the breaking pr?

Reply:
https://github.com/Azure/azure-rest-api-specs/pull/24163

Reply:
This breaking is acceptable and un-aviodeable.